README.md

```diff
@@ -129,6 +129,10 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://bigml.com/ - Hosted machine learning algorithms. Unlimited free tasks for development, limit of 16MB data per task
   * https://www.activestate.com/stackato/ - Enterprise-hardened Cloud Foundry PaaS from ActiveState, for private, public and hybrid cloud, free up to 20GB
 
+## Web Hosting
+
+  * https://www.simplybuilt.com - SimplyBuilt offers free website building and hosting for open source projects (http://www.simplybuilt.com/explore/free-websites-for-open-source-projects). Simple alternative to GitHub Pages.
```

Comment:
this looks nice but yeah I'd like it in a different category, it's getting a bit silly how many categories there are :)

But I dont see anywhere else so will just go with this